packages/box/box.ts

```diff
@@ -24,9 +24,20 @@ const defaultPath = "git@github.com:trufflesuite/truffle-init-default";
 export const normalizeSourcePath = (url = defaultPath) => {
   // Process filepath resolution
   //
-  if (url.startsWith(".") || url.startsWith("/") || url.startsWith("~")) {
-    debug({ in: url, out: path.normalize(url) });
-    return path.resolve(path.normalize(url));
+  const normalizedPath = path.normalize(url);
+  const resolvedPath = path.resolve(normalizedPath);
+  const pathExists = fse.existsSync(resolvedPath);
+
+  debug('---');
+  debug({ 
+    normalizedPath, 
+    resolvedPath,
+    pathExists
+  });
+
+  if(pathExists) {
```

Comment:
Can you explain why you added this check?

Reply:
Previously this function determined if the path was a file-system path to be used by `truffle unbox` by checking if if the path starts with `.`,`~` or '/'. I changed the check to be: does this path exist on the file-system to handle windows
paths. There should be more unit tests for this code path.

Reply:
Hm, OK.  So it's not a problem that if the path takes the form of a file system path, but doesn't actually exist, it won't take that branch?  Because obviously that's different.

Reply:
Yes, if it doesn't exist as a path, the next checks would be url like tests and finally a default error case.

Reply:
OK, so under this PR it'll hit the default error case, whereas previously it would have done what exactly?

Reply:
Ok, I'm going to move this to a draft because the unboxing piece needs more work. Previously it gave more meaningful and actionable errors (apart from network shenanigans). This PR breaks that. 

@haltman-at thanks for the push back on this. I think I have a better way forward that would keep meaningful errors. Most of the unboxing branches on isURLPathLike logic based on unix fs assumptions. I think we can more accurately determine if a url is a VCS like url (https:, git:)